acceptance-tests/src/main/java/io/blueocean/ath/SmartWebElement.java

```diff
@@ -66,78 +46,25 @@ protected WebDriver getDriver() {
     }
 
     /**
-     * Gets elements
-     * @return the elements found
+     * @return a FluentWait with default polling / timeout / exception handling
      */
-    public List<WebElement> getElements() {
-        return new FluentWait<>(getDriver())
-            .pollingEvery(100, TimeUnit.MILLISECONDS)
-            .withTimeout(DEFAULT_TIMEOUT, TimeUnit.MILLISECONDS)
-            .ignoring(NoSuchElementException.class)
-            .until(ExpectedConditions.numberOfElementsToBeMoreThan(by, 0));
-    }
-
-    /**
-     * Gets the first matching element
-     * @return see description
-     */
-    public WebElement getElement() throws NoSuchElementException {
-        List<WebElement> elements = getElements();
-        if (elements == null || elements.isEmpty()) {
-            throw new NoSuchElementException("Nothing matched for: " + expr);
-        }
-        if (elements.size() > 1) {
-            throw new NoSuchElementException("Too many elements returned for: " + expr);
-        }
-        return elements.get(0);
-    }
-
-    /**
-     * Iterates over all found elements with the given function
-     * @param fn to perform on the elements
-     */
-    public void forEach(java.util.function.Consumer<WebElement> fn) {
-        for (WebElement e : getElements()) {
-            fn.accept(e);
-        }
-    }
-
-    /**
-     * Determines if the element is visible
-     * @return true if visible, false if not
-     */
-    public boolean isVisible() {
```

Comment:
i guess this is just duplicate noise, but i really don't like `isDisplayed` this mirrors jQuery's terminology